cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/UploadApplicationPayloadHttpMessageConverter.java

```diff
@@ -80,20 +74,4 @@ private void setOutputContentType(MediaType contentType, HttpOutputMessage outpu
         }
     }
 
-    private void writeApplicationZipToFile(InputStream inputStream) {
-        // for testing/debugging purposes, write the zip file being uploaded to a path specified
-        // in the following environment variable
-        String uploadFilePath = System.getenv("CF_APP_UPLOAD_FILE");
```

Comment:
check if we use CF_APP_UPLOAD_FILE anywhere or if we can scrap the testing/debugging functionality altogether.

Reply:
We do not set CF_APP_UPLOAD_FILE anywhere. Unit tests don't set this env.